src/Styles/TODOList.css

```diff
@@ -0,0 +1,110 @@
+.todoListMainView {
```

Comment:
We are using tailwind css so we don't need external css files as much, you should check it out, makes life easier. https://tailwindcss.com/docs/utility-first

Reply:
yes I understand. But I still need my TODOList.css file to style some of the elements. But I did use a lot of the pre-styled components from material tailwind CSS

Reply:
I don’t mean the pre built components from material-tailwind but tailwindcss itself. It is a way of styling css, please check the link I sent above.

Reply:
I saw it. I personally kinda of prefer the CSS sheet approach to that. Each html tag ends up having so much style info embedded in it. Also, this can be updated later on too. So this should not be a blocker?

Reply:
It is not a blocker but our code style and UI will not be uniform if everyone else is using tailwindcss. Up to you when you would like to update, just letting you know. Your blocker is that merge conflict, nothing can be approved with it.

Reply:
Ok. I will migrate over to this style over time then